src/components/layout/Header.tsx

```diff
@@ -7,7 +7,7 @@ import { colors } from '../../theme/styleVars';
 import { ReactComponent as Logo } from '../../images/logoPlain.svg';
 
 const Header = () => (
-  <WhiteBackNav className="white-back nav" expand="lg" sticky="top">
+  <WhiteBackNav className="white-back nav container" expand="lg" sticky="top">
```

Comment:
Btw: adding a container here is fine, since WhiteBackNav is already extending another component. I also just noticed it, but let's start alphabetizing classNames in order: `container nav white-back`